terracumber_config/tf_files/aws_mirror.tf

```diff
@@ -0,0 +1,186 @@
+  default = "https://github.com/uyuni-project/uyuni.git"
+}
+
+variable "CUCUMBER_BRANCH" {
+  type = "string"
+  default = "master"
+}
+
+variable "CUCUMBER_RESULTS" {
+  type = "string"
+  default = "/root/spacewalk/testsuite"
+}
+
+variable "MAIL_SUBJECT" {
+  type = "string"
+  default = "Results Uyuni-Master $status: $tests scenarios ($failures failed, $errors errors, $skipped skipped, $passed passed)"
+}
+
+variable "MAIL_TEMPLATE" {
+  type = "string"
+  default = "../mail_templates/mail-template-jenkins.txt"
+}
+
+variable "MAIL_SUBJECT_ENV_FAIL" {
+  type = "string"
+  default = "Results Uyuni-Master: Environment setup failed"
+}
+
+variable "MAIL_TEMPLATE_ENV_FAIL" {
+  type = "string"
+  default = "../mail_templates/mail-template-jenkins-env-fail.txt"
+}
+
+variable "MAIL_FROM" {
+  type = "string"
+  default = "mnoel@suse.de"
+}
+
+variable "MAIL_TO" {
+  type = "string"
+  default = "mnoel@suse.de"
+}
+
+// sumaform specific variables
+variable "SCC_USER" {
+  type = "string"
+}
+
+variable "SCC_PASSWORD" {
+  type = "string"
+}
+
+variable "GIT_USER" {
+  type = "string"
+  default = null // Not needed for master, as it is public
+}
+
+variable "GIT_PASSWORD" {
+  type = "string"
+  default = null // Not needed for master, as it is public
+}
+
+variable "REGION" {
+  type = "string"
+  default = "eu-central-1"
+}
+
+variable "AVAILABILITY_ZONE" {
+  type = "string"
+  default = "eu-central-1a"
+}
+
+variable "KEY_FILE" {
+  type = "string"
+  default = "/home/jenkins/.ssh/testing-suma.pem"
+}
+
+variable "KEY_NAME" {
+  type = "string"
+  default = "testing-suma"
+}
+
+variable "ACCESS_KEY" {
+  type = "string"
+  default = null
+}
+
+variable "SECRET_KEY" {
+  type = "string"
+  default = null
+}
+
+variable "SERVER_REGISTRATION_CODE" {
+  type = "string"
+  default = null
+}
+
+variable "PROXY_REGISTRATION_CODE" {
+  type = "string"
+  default = null
+}
+
+provider "aws" {
+  region     = var.REGION
+  access_key = var.ACCESS_KEY
+  secret_key = var.SECRET_KEY
+}
+
+module "base" {
+  source = "./modules/base"
+
+  cc_username = var.SCC_USER
+  cc_password = var.SCC_PASSWORD
+  server_registration_code = var.SERVER_REGISTRATION_CODE
+  proxy_registration_code = var.PROXY_REGISTRATION_CODE
+  name_prefix  = "uyuni-mu-aws-"
+  //  mirror = "ip-172-16-1-50.eu-central-1.compute.internal"
+  provider_settings = {
+    availability_zone = var.AVAILABILITY_ZONE
+    region            = var.REGION
+    ssh_allowed_ips   = ["202.180.93.210", "65.132.116.252","195.135.221.27"]
+    key_name          = var.KEY_NAME
+    key_file          = var.KEY_FILE
+  }
+}
+
+
+module "mirror" {
+  source = "./modules/mirror"
+  base_configuration = module.base.configuration
+  disable_cron = true
+  provider_settings = {
+    public_instance = true
+  }
+  volume_provider_settings = {
+    # uncomment the following line if you want to reuse an data disk snapshot
+    //    volume_snapshot_id = data.aws_ebs_snapshot.data_disk_snapshot.id
+  }
+}
```

Comment:
Could you explain why we need a new terraform file for this module?
Can't we include this module inside the other terraform file deploying in AWS?